gapis/api/vulkan/framegraph.go

```diff
@@ -461,12 +593,12 @@ func (API) GetFramegraph(ctx context.Context, p *path.Capture) (*api.Framegraph,
 }
 
 // updateDependencies establishes dependencies between renderpasses.
-func updateDependencies(rpInfos []*renderpassInfo, dependencyGraph dependencygraph2.DependencyGraph) {
+func updateDependencies(workloadInfos []*workloadInfo, dependencyGraph dependencygraph2.DependencyGraph) {
 	// isInsideRenderpass: node -> renderpass it belongs to.
 	isInsideRenderpass := map[dependencygraph2.NodeID]uint64{}
-	for _, rpInfo := range rpInfos {
-		for _, n := range rpInfo.nodes {
-			isInsideRenderpass[n] = rpInfo.id
+	for _, wlInfo := range workloadInfos {
+		for _, n := range wlInfo.nodes {
+			isInsideRenderpass[n] = wlInfo.id
```

Comment:
I'm wondering whether this part still works as expected? After the nodes becomes more general and not only being renderpass node?

Reply:
Good catch, thanks! The logic is still valid, but the variable names and comments do no match anymore, indeed. I've update them to reflect that this now operates on workloads (and not only renderpasses).